tests/specs/issues/issue0413.txt

```diff
@@ -0,0 +1,17 @@
+~~ operatorPosition: maintain ~~
+== should format maintaining the operator position ==
+export function getIterator(iterable: I): Iterator<
+    I extends ReadonlyESMap<infer K, infer V> ? [K, V] :
+    I extends ReadonlySet<infer T> ? T :
+    I extends readonly (infer T)[] ? T :
+    I extends undefined ? undefined :
+    never>;
+
+[expect]
+export function getIterator(iterable: I): Iterator<
+    I extends ReadonlyESMap<infer K, infer V> ? [K, V] :
+        I extends ReadonlySet<infer T> ? T :
+        I extends readonly (infer T)[] ? T :
+        I extends undefined ? undefined :
+        never
```

Comment:
I like this not indenting in this case. I'll open up a separate PR after this one for that.

Reply:
Actually, this is done for when there's multiple type parameters so I'm going to leave it as-is.